src/main/java/com/epam/tc/api/hw3/service/RestBoardAssertion.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.tc.api.hw3.service;
+
+import com.epam.tc.api.hw3.dto.BoardDTO;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+
+public class RestBoardAssertion {
+
+    public void assertBoardName(BoardDTO boardDTO, String boardName) {
+
+        Assertions
+            .assertThat(boardDTO.getName())
+            .as(String.format("The Board name actual: %s \n The Board name is expected: %s",
+                boardDTO.getName(), boardName))
+            .isEqualTo(boardName);
+    }
+
+    public void assertBoardId(BoardDTO boardDTO) {
+
+        Assertions
+            .assertThat(boardDTO.getId())
+            .as("Board id is not valid")
+            .isNotEqualTo(null);
+    }
+
+    public void assertBoardDeleted(Response response) {
+
+        Assertions.assertThat(response.path("_value") == null).isEqualTo(true);
```

Comment:
This assertion looks a bit unclear, I would propose to add another step to the scenario - get the board and verify that it's not found. Or if you want first check that DELETE request was successful (verify status code)